### ISSUE TITLE
algoliaSettings is not defined

### ISSUE THREAD

**Issue by rahilwazir** [algolia/algoliasearch-wordpress#100]
I've installed and configured the algolia plugin, now whenever I search for term, it redirects me to search page and nothing outputs.

I'm getting and following error on search page template:

```
Uncaught ReferenceError: algoliaSettings is not defined
  (anonymous function) @ template.js?ver=4.5.2:9
  l @ algoliaBundle.min.js:2
  fireWith @ algoliaBundle.min.js:2
  ready @ algoliaBundle.min.js:2
  p @ algoliaBundle.min.js:1
```

**Comment by maxiloc** [comment]
@rahilwazir we are currently working on a new version of the plugin.

There is currently a private beta program running, if you are interested please send us an email to support+wordpress@algolia.com

**Comment by rayrutjes** [comment]
Hi @rahilwazir, this should be fixed in the latest version of the plugin, 0.2.5 at the time of writing this.
Please note however that this new version is not compatible with 0.0.1 and that the features have changed.

The [documentation](https://community.algolia.com/wordpress/installation.html) will give you an overview of what can be achieved with the new plugin.

We also have a list of [Frequently asked questions](https://community.algolia.com/wordpress/frequently-asked-questions.html)

I am closing this issue as it relates to version 0.0.1, but if you think it still applies to the new version, please create a new issue.

Also, if you have a question that the documentation doesn't cover, or if you want to report a problem or simply suggest a feature, please re-open an issue.
